snapcraft/_store.py

```diff
@@ -348,6 +354,24 @@ def download(snap_name, channel, download_path, arch):
                 snap_name, download_path))
 
 
+def history(snap_name, series, arch):
+    store = storeapi.StoreClient()
+
+    with _requires_login():
+        history = store.get_snap_history(snap_name, series, arch)
+
+    parsed_revisions = [
+        (rev['revision'], rev['timestamp'], rev['arch'], rev['version'],
+         _get_text_for_current_channels(
+            rev['channels'], rev['current_channels']))
+        for rev in history]
+    tabulated_revisions = tabulate(
+        parsed_revisions,
+        headers=['Rev.', 'Uploaded', 'Arch', 'Version', 'Channels'],
+        tablefmt="plain")
```

Comment:
oh, we have in a guy in our team that told us to always use single quotes unless it is a docstring ;-)